wpilibNewCommands/src/main/native/cpp/frc2/command/CommandScheduler.cpp

```diff
@@ -236,6 +251,12 @@ void CommandScheduler::Run() {
       Schedule({subsystem.getSecond().get()});
     }
   }
+
+  m_watchdog.Disable();
+  if (m_watchdog.IsExpired()) {
+    std::cout << "CommandScheduler loop overrun." << std::endl;
```

Comment:
Use wpi::outs() from <wpi/raw_ostream.h> instead. It's more efficient. Also, you can do this in the Watchdog lambda instead of you want it to print when the watchdog expires. At the moment, you're passing an empty lambda.

Reply:
I moved the primary loop overrun message to the lambda and I am using `wpi::outs()` now. However, I still kept the call to `PrintEpochs()` the way it was since that is [how it's done](https://github.com/wpilibsuite/allwpilib/blob/master/wpilibc/src/main/native/cpp/IterativeRobotBase.cpp#L186) in IterativeRobotBase as well.